nixos/modules/virtualisation/vmware-host.nix

```
@@ -0,0 +1,160 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.virtualisation.vmware.host;
+  wrapperDir = "/run/vmware/bin"; # Perfectly fits as /usr/local/bin
+  parentWrapperDir = dirOf wrapperDir;
+  vmwareWrappers = # Needed as hardcoded paths workaround
+    let mkVmwareSymlink =
+      program:
+      ''
+        ln -s "${config.security.wrapperDir}/${program}" $wrapperDir/${program}
+      '';
+    in
+    [
+      (mkVmwareSymlink "pkexec")
+      (mkVmwareSymlink "mount")
+      (mkVmwareSymlink "umount")
+    ];
+in
+{
+  options = with lib; {
+    virtualisation.vmware.host = {
+      enable = mkEnableOption "Enable VMWare host virtualisation";
+      package = mkOption {
+        type = types.package;
+        default = pkgs.vmware-workstation;
+        defaultText = literalExpression "pkgs.vmware-workstation";
+        description = ''
+          VMWare host virtualisation package to use
+          <note><para>
+          <literal>vmware-vmx</literal> will likely cause CPU hogging kcompactd0 process to appear because of Transparent Hugepages
+          Use <literal>boot.kernelParams = [ "transparent_hugepage=never" ];</literal> to disable them
+          If that didn't work disable <literal>TRANSPARENT_HUGEPAGE</literal> and <literal>COMPACTION</literal> config option when compiling kernel
+          </para></note>
+        '';
+      };
+      extraPackages = mkOption {
+        type = with types; listOf package;
+        default = with pkgs; [ ];
+        defaultText = literalExpression ''
+          with pkgs; [ ]
+        '';
+        description = "Extra packages to be used with VMWare host.";
+        example = "pkgs.ntfs3g";
```

```suggestion
        default = [];
        description = "Extra packages to be used with VMWare host.";
        example = "with pkgs; [ ntfs3g ]";
```

You can leave the default as an empty list and remove the default text. Also, the example provided would be incorrect as it is not a list of packages, so that would also fix it.